src/DocumentationExamples/DocumentationExamples.csproj

```diff
@@ -17,7 +17,7 @@
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.9.0" />
     <PackageReference Include="xunit.runner.visualstudio" Version="2.4.1" />
     <PackageReference Include="xunit" Version="2.4.1" />
-    <PackageReference Include="PublicApiGenerator" Version="8.1.0" />
+    <PackageReference Include="PublicApiGenerator" Version="10.0.0-beta3" />
```

Comment:
10.0.0-beta3 has a bug with `readonly` structs, it was fixed in beta4

Reply:
This doesn't affect Shouldly, but I added a commit updating to 10.0.0.